### PR TITLE
Feat/add  testing thresholds

### LINKED ISSUE #66: Turn on test coverage
Currently we're not testing for coverage. We should turn it on and use appropriate thresholds to avoid failing the build but also preventing coverage from decreasing. 

Let's also configure this in .taprc instead of CLI flags

### PR DESCRIPTION
**Pull request by TomasTM**
closes #66

### CODE REVIEW

**Comment by TomasTM** [comment]
@simoneb can you please give me some early feedback here? thanks

File: .taprc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+ts: false
+jsx: false
+flow: false
```
Comment (simoneb) [inline .taprc:3]: I don't think these are needed, are they?

Comment (TomasTM) [reply .taprc]: No you're right! Removed them

**Comment by simoneb** [comment]
link issue please

**Comment by TomasTM** [comment]
> link issue please

@simoneb issue linked

File: .taprc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+coverage: true
```
Comment (simoneb) [inline .taprc:1]: isn't coverage true by default?

Comment (TomasTM) [reply .taprc:1]: I don't think so, what is true by default is `check-coverage`. let me confirm that

Comment (TomasTM) [reply .taprc]: Yup! Not necessary indeed: https://node-tap.org/docs/coverage/ - removed

Comment (simoneb) [reply .taprc]: okay but can you see the level of additional work that you not looking into this beforehand causes? can you please put more attention to these fairly straightforward details?

**Review by simoneb — APPROVED** [review]
why has package-lock changed?

File: .taprc (hunk 2 above)
Comment (TomasTM) [reply .taprc]: To be fair, this was a draft for early feedback, but nevertheless - point noted 👍

**Comment by TomasTM** [comment]
> why has package-lock changed?

It changed on npm install, but to be on the safe side I'll the changes

**Review by simoneb — APPROVED** [review]

**Pull request merged**